tsdb/engine/tsm1/cache.go

```diff
@@ -461,13 +469,13 @@ func (c *Cache) updateMemSize(b int64) {
 }
 
 // Update the snapshotsCount and the diskSize levels
-func (c *Cache) updateSnapshot() {
+func (c *Cache) updateSnapshots() {
```

Comment:
This was renamed to `updateSnapho()t` because there is only ever one snapshot now.